src/applications/common/schemaform/review/ReviewCollapsibleChapter.jsx

```diff
@@ -127,6 +127,9 @@ export default class ReviewCollapsibleChapter extends React.Component {
               pageData = form.data;
               fullPageKey = page.pageKey;
             }
+            if (page.hideOnReview) {
```

Comment:
There is already a hideOnReview property for properties, is it helpful or confusing to use the same name for pages? Would it be helpful to specify that this is for array pages with custom field/widgets by including the word "array"?

Reply:
I think having the same name is fine if it's doing the same thing, but I'm having a hard time wrapping my head about what's happening here. If we're just trying to exclude the page from being displayed, would it be clearer to put a filter it out of `expandedPages` before calling `map`?